ws-server/src/tests.rs

```diff
@@ -122,22 +122,22 @@ async fn server_with_context() -> SocketAddr {
 
 	rpc_module
 		.register_method("should_err", |_p, ctx| {
-			let _ = ctx.err().map_err(|e| CallError::Failed(e.into()))?;
+			let _ = ctx.ok().map_err(CallError::Failed)?;
```

Comment:
Is the swap from `err` to `ok` here intentional?

Reply:
copy-past nit but those are fixed now, good catch